examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -222,6 +222,32 @@ CHIP_ERROR PairingCommand::AddWiFiNetwork()
     return mCluster.AddWiFiNetwork(successCallback, failureCallback, mSSID, mPassword, kBreadcrumb, kTimeoutMs);
 }
 
+chip::ByteSpan PairingCommand::GetThreadNetworkId()
+{
+    // For Thread devices the networkId is the extendedPanId and it is
+    // part of the dataset defined by OpenThread
+
+    ByteSpan threadNetworkId;
+
+    Thread::OperationalDataset dataset;
+    CHIP_ERROR err = dataset.Init(mOperationalDataset);
+    if (err == CHIP_NO_ERROR)
+    {
+        err = dataset.GetExtendedPanId(mExtendedPanId);
+        if (err == CHIP_NO_ERROR)
+        {
+            return ByteSpan(mExtendedPanId, sizeof(mExtendedPanId));
+        }
+    }
+
+    if (threadNetworkId.empty())
```

Comment:
I you reach this code  isn't `threadNetworkId` always empty  ? Maybe you may want to rewrite this code such as: 
```
chip::ByteSpan PairingCommand::GetThreadNetworkId()
{
     // For Thread devices the networkId is the extendedPanId and it is
     // part of the dataset defined by OpenThread

     Thread::OperationalDataset dataset;
     CHIP_ERROR err = dataset.Init(mOperationalDataset);
     if (err == CHIP_NO_ERROR)
     {
         err = dataset.GetExtendedPanId(mExtendedPanId);
         if (err == CHIP_NO_ERROR)
         {
             return ByteSpan(mExtendedPanId, sizeof(mExtendedPanId));
         }
     }

     return ByteSpan();
}
```

Reply:
Addressed all reviews but it got autoclosed during force push, had to open new one because Reopen button is disabled.

https://github.com/project-chip/connectedhomeip/pull/8245